hack/install.yaml

```diff
@@ -68,30 +68,42 @@ spec:
           type: object
         spec:
           properties:
+            assemblyPhase:
+              type: string
             componentKinds:
               items:
-                properties:
-                  group:
-                    type: string
-                  kind:
-                    type: string
                 type: object
               type: array
             description:
               type: string
-            assemblyPhase:
-              type: string
-              enum:
-              - Pending
-              - Succeeded
-              - Failed
             info:
               items:
                 properties:
                   name:
                     type: string
                   value:
                     type: string
+                  valueFrom:
+                    properties:
+                      configMapKeyRef:
+                        type: object
+                      ingressRef:
+                        properties:
+                          host:
+                            type: string
+                          path:
+                            type: string
+                        type: object
+                      secretKeyRef:
```

Comment:
@kow3ns I'm using the latest kubebuilder, but it doesn't seem to be able to pull in the definitions of core structs here.